opencga-app/src/main/java/org/opencb/opencga/app/migrations/v2_4_3/catalog/ShortenIndexedFieldVariables.java

```diff
@@ -0,0 +1,83 @@
+package org.opencb.opencga.app.migrations.v2_4_3.catalog;
+
+import com.mongodb.client.model.Projections;
+import com.mongodb.client.model.UpdateOneModel;
+import org.bson.Document;
+import org.opencb.commons.datastore.mongodb.GenericDocumentComplexConverter;
+import org.opencb.opencga.catalog.db.mongodb.MongoDBAdaptorFactory;
+import org.opencb.opencga.catalog.migration.MigrationTool;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static com.mongodb.client.model.Filters.eq;
+
+public class ShortenIndexedFieldVariables extends MigrationTool {
+
+    @Override
+    protected void run() throws Exception {
+
+        migrateCollection(Arrays.asList(
```

Comment:
What about using this `$rename` ?
https://www.mongodb.com/docs/manual/reference/operator/update/rename/